3p/ampcontext.js

```diff
@@ -100,7 +100,7 @@ export class AbstractAmpContext {
     /** @protected {!IframeMessagingClient} */
     this.client_ = new IframeMessagingClient(win);
     this.client_.setHostWindow(this.getHostWindow_());
-    this.client_.setSentinel(dev().assertString(this.sentinel));
```

Comment:
Its an integration question.  Sentinel is expected to be included in the frame.  If its not, it implies the library was used incorrectly (in the wrong location).  If we leave as dev, it gets compiled out and no one is notified.